src/components/DocumentationTopic.vue

```diff
@@ -643,7 +643,7 @@ export default {
         seeAlsoSections,
         variantOverrides,
         variants = [],
-      } = json;
+      } = json || {};
```

Comment:
Same question as above: is there ever a realistic scenario where `extractProps` is called with a null/undefined value?